packages/editor/src/components/mobile/bottom-sheet/cell.native.js

```diff
@@ -7,85 +7,132 @@ import { TouchableOpacity, Text, View, TextInput, I18nManager } from 'react-nati
  * WordPress dependencies
  */
 import { Dashicon } from '@wordpress/components';
+import { Component } from '@wordpress/element';
 
 /**
  * Internal dependencies
  */
 import styles from './styles.scss';
+import platformStyles from './cellStyles.scss';
```

Comment:
@etoledom I have added here `.scss` so that test can pass.

Reply:
Thank you!